src/ae.c

```diff
@@ -503,3 +511,9 @@ void aeSetBeforeSleepProc(aeEventLoop *eventLoop, aeBeforeSleepProc *beforesleep
 void aeSetAfterSleepProc(aeEventLoop *eventLoop, aeAfterSleepProc *aftersleep) {
     eventLoop->aftersleep = aftersleep;
 }
+
+/*This function allows setting a custom poll procedure to be used by the event loop
+ * The custom poll procedure, if set, will be called instead of the default aeApiPoll */
```

Comment:
Good documentation, thanks.

Nit: punctuation

```suggestion
/* This function allows setting a custom poll procedure to be used by the event loop.
 * The custom poll procedure, if set, will be called instead of the default aeApiPoll. */
```

Reply:
Done